ui/js/directives/perf/perf.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
+ 
+'use strict';
+
+perf.directive("testDataChooser", function() {
```

Comment:
If this directive could conceivably be used elsewhere later in treeherder (?), I wonder if it should live in our existing treeherder [/directives](https://github.com/mozilla/treeherder/tree/master/ui/js/directives) dir and be accessed there. Perhaps check with Will to see.